core/block_validator.go

```diff
@@ -147,10 +148,10 @@ func (v *BlockValidator) ValidateState(block *types.Block, statedb *state.StateD
 // the gas allowance.
 func CalcGasLimit(parentGasUsed, parentGasLimit, gasFloor, gasCeil uint64) uint64 {
 	// contrib = (parentGasUsed * 3 / 2) / 1024
-	contrib := (parentGasUsed + parentGasUsed/2) / params.GasLimitBoundDivisor
+	contrib := (parentGasUsed + parentGasUsed/2) / extras.GasLimitBoundDivisor
```

Comment:
GasLimitBoundDivisor, MinGasLimit and MaxGasLimit were moved from params to params/extras. In coreth it's at plugin/evm/upgrades/ap0 let me know if you think there is a more appropriate location for them.
They are relocated to avoid a cycle dependency between params and plugin/evm/header